servers/rendering/renderer_scene_cull.cpp

```diff
@@ -608,8 +608,19 @@ void RendererSceneCull::instance_set_base(RID p_instance, RID p_base) {
 	instance->base = RID();
 
 	if (p_base.is_valid()) {
+		RS::InstanceType old_base = instance->base_type;
 		instance->base_type = RSG::utilities->get_base_type(p_base);
 
+		if (old_base == RS::INSTANCE_NONE && ((1 << instance->base_type) & RS::INSTANCE_GEOMETRY_MASK) 
+			&& instance->indexer_id.is_valid()) {
```

Comment:
```suggestion
		if (old_base == RS::INSTANCE_NONE && ((1 << instance->base_type) & RS::INSTANCE_GEOMETRY_MASK) && instance->indexer_id.is_valid()) {
```

Reply:
Somehow the clang-format in my local machine doesn't always seems to work, sorry. 
I will deal with formatting problems after reviews think this method makes sense.

Reply:
Thanks！